os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -204,6 +213,31 @@ static void rtl8730e_control_backlight(uint8_t level)
 #endif
 }
 
+static void rtl8730e_lcd_init_screen(int xres, int yres)
+{
+	int len = xres * yres * 2;
+	lcd_data = malloc(len);
+	if (lcd_data == NULL) {
+		lcddbg("malloc failed for lcd data : %d\n", len);
+		rtl8730e_control_backlight(CONFIG_LCD_MAXPOWER);
+		return;
+	}
+	for (int i = 0; i < len; i += 2) {
+		lcd_data[i] = (INITCOLOR & 0xFF00) >> 8;
+		lcd_data[i + 1] = INITCOLOR & 0x00FF;
+	}
+	rtl8730e_lcd_put_area(lcd_data, 0, 0, xres, yres);
```

Comment:
In here you are using put area, which is in video mode to update black screen during initialization.
As I heard, the request seems only rendering one time, and when UIFW done initializing, it will start to update the frame.
Thus, the flow here should be changed to update the pixels by command mode, instead of video mode (ie. Use 0x2C command, and throw the buffer into the msg).

Reply:
```
msg_col.channel = 0x2C;
msg_col.type = MIPI_DSI_DCS_LONG_WRITE;
msg_col.tx_buf = lcd_data;
msg_col.tx_len = len;
msg_col.flags = 0;
```

Reply:
What will be benefit of using command mode for this instead of video mode?
We have to malloc and free buffer in both cases.

Reply:
No need to wait time to free. With the video mode, we must get the init command by upper layer. If it's missing in upper layer, it will be a leak. It means it has a dependency. But if we use the command mode, we can free it after sending. No need to get dependency with upper layer. It's what I understand.

Reply:
@edwakuwaku 
We are finding strange behaviour when using cmd mode for sending data during initialization. Please confirm
1. After sending data via cmd mode, LCD does not show content till we change to video mode.
2. LCD shows noise, when we free the memory(lcd_data here) for cmd mode data
msg_col.tx_buf = lcd_data;
3. Is there any need to initalize the memory pointer(lcdc_init_struct.layerx[idx].LCDC_LayerImgBaseAddr) even when we are sending data via cmd mode

Reply:
Hi @anjana348,
If you want to use cmd mode to update the frame, you need to program the LCM to cmd mode.  
{0xB0, {0x00, 0x00, 0x00, 0xF1}} before using cmd mode to send the image buffer.

Reply:
@zhongnuo-tang 
I tried below sequence, but no data refresh on LCD
MIPI CMD Mode
Send {0xB0, {0x00, 0x00, 0x00, 0xF1}} 
Send 0x2A (coordinates), 0x2B (coordinates)
Send 0x2C (lcd buffer)
Did you verify this working? Or do I need to change anything else?